Makefile

```diff
@@ -341,7 +341,7 @@ validate: gofmt .gitvalidation validate.completions
 
 build-all-new-commits:
 	# Validate that all the commits build on top of $(GIT_BASE_BRANCH)
-	git rebase $(GIT_BASE_BRANCH) -x make
+	git rebase $(GIT_BASE_BRANCH) -x "make | awk '{ print strftime(\"%H:%M:%S\"), $0 }'"
```

Comment:
You can safely drop the double quotes

Reply:
As in: the outside ones; then the inside ones (%H:etc) don't need to be backslashed

Reply:
Grumble. You have to double-dollar it: `$$0`. (I thought `make` would respect the single-quotes; it doesn't)

Reply:
Without outside quotes, we get one time print for the entire git rebase command - is that what you intended?

Reply:
`awk` filters line by line, so each line gets its own timestamp (assuming I/O gets flushed properly on this system)